osde2e/aws_vpce_operator_runner_test.go

```diff
@@ -1,27 +1,27 @@
+// THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
 //go:build integration
 // +build integration
 
-package exampleaddontestharness
+package osde2e_tests
```

Comment:
Package names cannot have underscores: https://go.dev/blog/package-names

Suggest something something like "osde2e:

Reply:
I followed the naming standard of aws_client package. I can rename it, if we want to go with this instead.

Reply:
pushed to boilerplate PR

Reply:
It doesn't look like this PR has the latest boilerplate? It's not matching here: https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh#L59